tests/test_base.py

```diff
@@ -69,6 +69,169 @@
+        np.testing.assert_raises(AttributeError, delta.drawImage)
+    except ImportError:
+        pass
+        
+    delta = galsim.DeltaFunction(flux=1)
+    try:
+        np.testing.assert_raises(AttributeError, delta.drawKImage)
+    except ImportError:
+        pass
+    
+    # Test illegal operations on DeltaFunction    
+    delta = galsim.DeltaFunction(flux=1)
+    try:
+        np.testing.assert_raises(AttributeError, delta.expand, 2.0)
+    except ImportError:
+        pass
+    
+    delta = galsim.DeltaFunction(flux=1)
+    try:
+        np.testing.assert_raises(AttributeError, delta.shear, g1=0.3, g2=0.1)
+    except ImportError:
+        pass
+    
+    delta = galsim.DeltaFunction(flux=1)
+    try:
+        np.testing.assert_raises(AttributeError, delta.dilate, 2.)
+    except ImportError:
+        pass   
+    
+    delta = galsim.DeltaFunction(flux=1)
+    try:
+        np.testing.assert_raises(AttributeError, delta.magnify, 2.)
+    except ImportError:
+        pass   
+
+    delta = galsim.DeltaFunction(flux=1)
+    try:
+        np.testing.assert_raises(AttributeError, delta.transform, dudx=1.0,
+                                 dudy=0.0, dvdx=0.0, dvdy=1.0)
+    except ImportError:
+        pass 
+    
+    delta = galsim.DeltaFunction(flux=1)
+    try:
+        np.testing.assert_raises(AttributeError, delta.lens, g1=0.3, g2=0.1, mu=2.)
+    except ImportError:
+        pass 
+    
+    delta = galsim.DeltaFunction(flux=1)
+    try:
+        np.testing.assert_raises(AttributeError, delta.rotate, 45 * galsim.radians)
+    except ImportError:
+        pass 
+    
+    # Test simple translation of DeltaFunction
+    delta = galsim.DeltaFunction(flux = test_flux)
+    delta2 = delta.shift(1.,2.)      # Shft the object in real space.
+    offcen = galsim.PositionD(1, 2)
+    np.testing.assert_equal(delta2.centroid(), offcen)
+    np.testing.assert_almost_equal(delta2.xValue(offcen), test_flux)
+    np.testing.assert_almost_equal(delta2.xValue(galsim.PositionD(0,0)), 0)
+    
+    # Check picklability
+    delta = galsim.DeltaFunction(flux=test_flux)
+    do_pickle(delta.SBProfile, lambda x: (x.getFlux(), x.getGSParams()))
+    do_pickle(delta)
+    do_pickle(delta.SBProfile)
+    
+@timer
+def test_deltaFunction_properties():
+    """Test some basic properties of the Delta function profile
+    """
+    delta = galsim.DeltaFunction(flux=test_flux)
+    # Check that we are centered on (0, 0)
+    cen = galsim.PositionD(0, 0)
+    np.testing.assert_equal(delta.centroid(), cen)
+    offcen = galsim.PositionD(1,1)
+    # Check Fourier properties
+    np.testing.assert_equal(delta.kValue(cen), (1+0j) * test_flux)
+    np.testing.assert_equal(delta.kValue(offcen), (1+0j) * test_flux)
+    import math
+    np.testing.assert_almost_equal(delta.xValue(cen), test_flux)
+    np.testing.assert_almost_equal(delta.xValue(offcen), 0)
+    np.testing.assert_array_less(1e100,delta.maxK())
+    np.testing.assert_array_less(1e100,delta.stepK())
```

Comment:
Rather than `assert_array_less` here, you can just do `assert delta.stepK() < 1.e100`.  Not everything has to go through the numpy testing module.  :)